src/views/bouquets/BouquetEditView.vue

```diff
@@ -69,49 +165,197 @@ onMounted(() => {
+                name="select-datasets"
+                v-model="selectedDataset.id"
+                :filter-results="false"
+                :min-chars="1"
+                :resolve-on-load="false"
+                :delay="0"
+                :searchable="true"
+                :options="search"
+                v-if="!isEditDesc"
+              />
+              <DsfrButton label="Valider" @click.prevent="onSubmitModal()" :secondary="true" />
+            </div>
 
-          <Tooltip
-            title="Objectif du bouquet"
-            name="tooltip__objectif"
-            text="Ajoutez ici l'ensemble des informations nécessaires à la compréhension, l'objectif et l'utilisation du bouquet. N'hésitez pas à indiquer la réglementation ou une documentation liée au bouquet."
-          />
-          <Tooltip
-            title="Utilisez du markdown pour mettre en forme votre texte"
-            name="tooltip__markdown"
-            text="* simple astérisque pour italique *<br/> ** double astérisque pour gras **<br/> # un dièse pour titre 1<br/> ## deux dièses pour titre 2<br/> *  astérisque pour une liste<br/> lien : [[https://exemple.fr]]"
+            
+        </div>
+
+        <div class="fr-grid-row">
+            <div class="fr-col-lg-6">
+              <DsfrInput
+                  v-model="urlData"
+                  placeholder="Url vers le jeu de données souhaité"
+                  label="Vous ne trouvez pas la donnée dans éÉcosphères ?"
+                  :label-visible="true"
+                />
+            </div>
+            <div class="fr-col-lg-3">
+              <DsfrButton
+                  icon="ri-delete-bin-line"
+                  label="Ajouter la donnée"
+                  :secondary="true"
+                  @click.prevent="onDeleteDataset(d.dataset.id)"
+                />
+            </div>
+        </div>
+        <hr/>
+        <div class="fr-grid-row">
+            <div class="fr-col-lg-7">
+              <DsfrInput
+                  v-model="libelle"
+                  class="fr-mb-3w"
+                  label="Libellé de la donnée"
+                  :label-visible="true"
+                />
+
+                <DsfrInput
+                  v-model="raison"
+                  class="fr-mb-3w"
+                  label="Raison d'utilisation dans ce bouquet*"
+                  hint="Et ici l’indice permanent"
+                  :label-visible="true"
+                  :is-textare="true"
+                />
+            </div>
+          </div>
+      
+        <DsfrInput
+          class="fr-mt-1w fr-mb-2w"
+          v-model="selectedDataset.description"
+          label="Description du jeu de données dans le bouquet"
+          placeholder="Pourquoi avez-vous ajouté ce jeu de données ?"
+          :label-visible="true"
+        />
+        <!-- <DsfrButton type="button" class="fr-mt-2w" label="Ajouter un jeu de données" @click.prevent="onSubmitModal()" /> -->
```

Comment:
Conservé pour plus tard ou à supprimer ?

Reply:
This one is old. Has been edited